sdk/eventhub/event-hubs/README.md

```diff
@@ -357,9 +357,26 @@ The Event Hubs library depends on the [rhea-promise](https://github.com/amqp/rhe
 
 ### Enable logs
 
-You can set the following environment variable to get the debug logs when using this library.
+This library depends on the [@azure/logger](https://www.npmjs.com/package/@azure/logger) package
+to support debug logs.
 
-- Getting debug logs from the Event Hubs SDK
+You can set the `AZURE_LOG_LEVEL` environment variable to one of the following values to enable logging to `stderr`:
+
+- verbose
+- info
+- warning
+- error
+
+You can also set the log level programatically by importing the `@azure/logger` package and calling the
+`setLogLevel` function with one of the log level values.
+
+When setting a log level either programatically or via the `AZURE_LOG_LEVEL` environment variable,
+any logs that are written using a log level equal to or less than the one you choose will be emitted.
+
+You can alternatively set the `DEBUG` environment variable to get logs when using this library.
+This can be useful if you also want to emit logs from the dependencies `rhea-promise` and `rhea` as well.
+
+- Getting debug logs from the Event Hubs SDK.
 
 ```bash
 export DEBUG=azure*
```

Comment:
Isnt this same as setting `AZURE_LOG_LEVEL` to `verbose`?
Should we include examples for `DEBUG` only if it provides a different value than `AZURE_LOG_LEVEL`?

Reply:
Updated the example.